inst/include/odeproblem.h

```diff
@@ -201,8 +201,8 @@ class odeproblem {
   int Npar;
   int Neq;
   int Istate; ///< istate value
-  double ss_tol;
-  double ss_n;
+  bool ss_fixed = false;
+  int ss_n = 500;
```

Comment:
Will `odeproblem` be ever constructed with these default values?

Reply:
These default values were removed.